Mainboard/REV20200920D/README.md

```diff
@@ -2,10 +2,20 @@ This folder contains schematics pdf, PCB pdf, bill of material (BOM), 3-D PCB, g
 
 ## How to order 10 PCB boards
 
-1. Place a new quote in https://www.pcbway.com/ 
+1. Start a quote from any of the following vendors.
+
+| Vendor Name             | Country      | Type |
+|----------------------|-------------|--|
+| [Screaming Circuits](https://www.screamingcircuits.com/)| US| Fabrication/Assembly |
+| [OshPark](https://oshpark.com/home)| US| Fabrication |
+| [Sunstone Circuits](https://www.sunstone.com/) | US | Fabrication |
+| [PCBWay](https://www.pcbway.com/) | China | Fabrication/Assembly |
+| [JLCPCB](https://jlcpcb.com/?_gl=1%2a1kjh7v7%2a_gcl_au%2aMzYwNzcxMDQwLjE3NDMxODc1MTI.%2a_ga%2aMjM3NTY3OTQzLjE3NDMxODc1MTI.%2a_ga_VB33MLCQS2%2aMTc0MzE4NzUxMi4xLjAuMTc0MzE4NzUxMi42MC4wLjA.%2a_ga_BZ8D96C9TK%2aMTc0MzE4NzUxMi4xLjAuMTc0MzE4NzUxMi42MC4wLjA.) | China | Fabrication/Assembly |
+
 
 2. Use the following table to set the pcb properties.
 
+## Fabrication Details
```

Comment:
It seems like this should be a subsubsection of "How to order" and not the same header level? Or maybe you mean something else.

It's also odd that we have a numbered list spanning multiple headers